config/envoy/300-gateway.yaml

```diff
@@ -0,0 +1,205 @@
+# Copyright 2020 The Knative Authors
```

Comment:
We should strive to keep the diff between the two files at 0, modulo the image reference. Is that not the case right now?

Reply:
Yeah, it would be better to keep the no diff. Updated. (I hope I didn't misunderstand your comment.)